source/guides/getting-started/quick-start.html.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Guides | Quick Start
+next_guide:
+  title: Interactors Introduction
+  url: /guides/interactors/introduction.html
+---
+
+# Quick Start
+
+This guide will teach you how to add BigTest to your React application.
+
+We'll cover:
+- Installing BigTest
+- Generating support files
+- Set Up
+- Writing some basic tests
+
+The steps outlined in this guide assume that you are using:
+
+- [ Mocha ](https://mochajs.org/)
+- Either Webpack or Parcel as your bundler.
+
+## Install Dependencies
+
+In your terminal, type in the following commands:
+
+```bash
+ yarn add --dev @bigtest/cli
+```
+
+```bash
+ yarn add --dev @bigtest/interactor
+```
+
+```bash
+ yarn add --dev @bigtest/react
+```
```

Comment:
Instead of listing these commands out separately, we can just have one command that can easily be copy and pasted:

```bash
yarn add --dev @bigtest/cli @bigtest/interactor @bigtest/react
```

Reply:
Ha, you are absolutely right. 💯

Reply:
I'd also intorduce with a sentence "First, let's install all of the packages you're going to need to use bigtest. In your terminal, type the following commands...."